server/sonar-server/src/main/java/org/sonar/server/computation/step/PersistSnapshotsStep.java

```diff
@@ -49,16 +46,14 @@
   private final DbClient dbClient;
   private final TreeRootHolder treeRootHolder;
   private final AnalysisMetadataHolder analysisMetadataHolder;
-  private final MutableDbIdsRepository dbIdsRepository;
   private final PeriodsHolder periodsHolder;
 
   public PersistSnapshotsStep(System2 system2, DbClient dbClient, TreeRootHolder treeRootHolder, AnalysisMetadataHolder analysisMetadataHolder,
```

Comment:
class could be renamed PersistAnalysisStep and step's description updated accordingly